db/migrate/20221129115950_devise_create_users.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[7.0]
```

Comment:
This migration is generated via `rails g devise User`. It is as generated except I have commented out the Recoverable options, and uncommented the Tackable ones.